cocos/3d/CCMesh.h

```diff
@@ -94,10 +95,39 @@ class CC_DLL Mesh : public Ref
     /**get per vertex size in bytes*/
     int getVertexSizeInBytes() const;
 
-    /**texture getter and setter*/
+    /**
+     * set diffuse texture
+     * @param texPath texture path
+     */
     void setTexture(const std::string& texPath);
+    /**
+     * set diffuse texture
+     * @param tex texture to be set
+     */
     void setTexture(Texture2D* tex);
```

Comment:
Got it.
I think it is better to add more clear comment to describe why keeping this function to set diffuse texture, and use setTexture(tex, usage) to set other type of textures.